sonar-project.properties

```diff
@@ -0,0 +1,36 @@
+# Organization and project keys are displayed in the right sidebar of the project homepage
+sonar.organization=owncloud-1
+sonar.projectKey=owncloud_core
+sonar.projectVersion=1.0
+sonar.host.url=https://sonarcloud.io
+
+# =====================================================
+#   Meta-data for the project
+# =====================================================
+
+sonar.links.homepage=https://github.com/owncloud/core
+sonar.links.ci=https://drone.owncloud.com/owncloud/core/
+sonar.links.scm=https://github.com/owncloud/core
+sonar.links.issue=https://github.com/owncloud/core/issues
+
+# =====================================================
+#   Properties that will be shared amongst all modules
+# =====================================================
+
+# Just look in these directories for code
+sonar.sources=.
+sonar.inclusions=apps/**,core/**,lib/**,ocs/**,settings/**
+
+# Pull Requests
+sonar.pullrequest.provider=GitHub
+sonar.pullrequest.github.repository=owncloud/core
+sonar.pullrequest.base=${env.SONAR_PULL_REQUEST_BASE}
+sonar.pullrequest.branch=${env.SONAR_PULL_REQUEST_BRANCH}
+sonar.pullrequest.key=${env.SONAR_PULL_REQUEST_KEY}
+
+# Properties specific to language plugins:
+sonar.php.coverage.reportPaths=results/clover-phpunit-php7.2-mariadb10.2.xml,results/clover-phpunit-php7.2-mariadb10.3.xml,results/clover-phpunit-php7.2-mariadb10.4.xml,results/clover-phpunit-php7.2-mariadb10.5.xml,results/clover-phpunit-php7.2-mysql5.5.xml,results/clover-phpunit-php7.2-mysql5.7.xml,results/clover-phpunit-php7.2-mysql8.0.xml,results/clover-phpunit-php7.2-oracle.xml,results/clover-phpunit-php7.2-postgres10.3.xml,results/clover-phpunit-php7.2-postgres9.4.xml,results/clover-phpunit-php7.2-sqlite-samba-samba.xml,results/clover-phpunit-php7.2-sqlite-samba.xml,results/clover-phpunit-php7.2-sqlite-scality.xml,results/clover-phpunit-php7.2-sqlite-sftp-sftp.xml,results/clover-phpunit-php7.2-sqlite-sftp.xml,results/clover-phpunit-php7.2-sqlite-webdav-webdav.xml,results/clover-phpunit-php7.2-sqlite-webdav.xml,results/clover-phpunit-php7.2-sqlite-windows-windows.xml,results/clover-phpunit-php7.2-sqlite-windows.xml,results/clover-phpunit-php7.2-sqlite.xml,results/clover-phpunit-php7.3-mariadb10.2.xml,results/clover-phpunit-php7.3-sqlite.xml,results/clover-phpunit-php7.4-mariadb10.2.xml,results/clover-phpunit-php7.4-sqlite.xml
```

Comment:
This comma-separated list needs to have names that match the coverage reports that should have been generated by each of the phpunit and phpintegration pipelines.

We can get it working with a hard-coded list, then see if we can think how to generate the list automagically within the sonar-analysis pipeline.

Reply:
done